modules/virt-creating-bridge-nad-cli.adoc

```diff
@@ -15,88 +14,56 @@ The network attachment definition must be in the same namespace as the pod or vi
 
 .Procedure
 
-. Create a new file for the network attachment definition in any local directory.
-The file must have the following contents, modified to match your
-configuration:
+. Create a network attachment definition manifest.
+The manifest must have the following contents, modified to match your configuration:
 +
 [source,yaml]
 ----
 apiVersion: "k8s.cni.cncf.io/v1"
 kind: NetworkAttachmentDefinition
 metadata:
-  name: a-bridge-network
+  name: <a-bridge-network> <1>
   annotations:
-    k8s.v1.cni.cncf.io/resourceName: bridge.network.kubevirt.io/br0 <1>
+    k8s.v1.cni.cncf.io/resourceName: bridge.network.kubevirt.io/br0 <2>
 spec:
   config: '{
     "cniVersion": "0.3.1",
-    "name": "a-bridge-network", <2>
+    "name": "<a-bridge-network>", <3>
     "plugins": [
       {
-        "type": "cnv-bridge", <3>
-        "bridge": "br0", <4>
-        "vlan": 1 <5>
+        "type": "cnv-bridge", <4>
+        "bridge": "br0", <5>
+        "vlan": 1 <6>
       },
       {
-        "type": "cnv-tuning" <6>
+        "type": "cnv-tuning" <7>
       }
     ]
   }'
 ----
-<1> If you add this annotation to your network attachment definition, your virtual machine instances
-will only run on nodes that have the `br0` bridge connected.
-<2> Required. A name for the configuration. It is recommended to match the configuration name to the `name` value of the network attachment definition.
-<3> The actual name of the Container Network Interface (CNI) plug-in that provides
-the network for this network attachment definition. Do not change this field unless
-you want to use a different CNI.
-<4> You must substitute the actual name of the bridge, if it is not `br0`.
-<5> Optional: The VLAN tag.
-<6> Required. This allows the MAC pool manager to assign a unique MAC address to the connection.
+<1> The name for the `NetworkAttachmentDefinition` object.
+<2> If you add this annotation to your network attachment definition, your virtual machine instances will only run on nodes that have the `br0` bridge connected.
```

Comment:
some bullets here explain the meaning (1,6) and some just a comment to it. 
Please make it the same for all. I suggest: [Required|Optional]: <short-description>. <extra comment>